projects/1.18/assets/butchercraft/lang/zh_cn.json

```
@@ -0,0 +1,201 @@
+{
+  "block.butchercraft.bbq_ribs_block": "炭烤排骨",
+  "block.butchercraft.beef_block": "生牛肉块",
+  "block.butchercraft.beef_wellington_block": "惠灵顿牛排",
+  "block.butchercraft.butcher_block": "案板",
+  "block.butchercraft.cooked_beef_block": "熟牛肉块",
+  "block.butchercraft.cooked_lamb_block": "熟羊肉块",
+  "block.butchercraft.cooked_pork_block": "熟猪肉块",
+  "block.butchercraft.cow_hide_carpet": "牛皮",
+  "block.butchercraft.drying_rack": "晾干架",
+  "block.butchercraft.haggis_block": "哈吉斯",
+  "block.butchercraft.hash_block": "肉丁土豆",
+  "block.butchercraft.jelly_black_block": "黑莓果冻",
+  "block.butchercraft.jelly_blue_block": "蓝莓果冻",
+  "block.butchercraft.jelly_brown_block": "可乐果冻",
+  "block.butchercraft.jelly_cyan_block": "潘趣果冻",
+  "block.butchercraft.jelly_green_block": "苹果果冻",
+  "block.butchercraft.jelly_grey_block": "神秘果冻",
+  "block.butchercraft.jelly_light_blue_block": "蓝色覆盆子果冻",
+  "block.butchercraft.jelly_light_grey_block": "火龙果果冻",
+  "block.butchercraft.jelly_lime_block": "酸橙果冻",
+  "block.butchercraft.jelly_magenta_block": "杂莓果冻",
+  "block.butchercraft.jelly_orange_block": "橘子果冻",
+  "block.butchercraft.jelly_pink_block": "草莓果冻",
+  "block.butchercraft.jelly_purple_block": "葡萄果冻",
+  "block.butchercraft.jelly_red_block": "樱桃果冻",
+  "block.butchercraft.jelly_white_block": "椰子果冻",
+  "block.butchercraft.jelly_yellow_block": "柠檬果冻",
+  "block.butchercraft.lamb_block": "生羊肉块",
+  "block.butchercraft.mashed_potato_gravy_block": "肉汁土豆泥",
+  "block.butchercraft.meat_hook": "挂肉钩",
+  "block.butchercraft.meat_pie": "肉馅饼",
```

```suggestion
  "block.butchercraft.meat_pie": "肉饼",
```
要么这里改成“肉饼”，要么后面的“肉饼切片”改成“肉馅饼切片”